packages/cli/src/presets/codemods/utils/transforms.tsx

```diff
@@ -0,0 +1,18 @@
+import path, { ParsedPath } from 'path';
+import glob from 'glob';
+
+import isDev from '../../../isDev';
+
+const basePath = isDev
+  ? path.join(process.cwd(), '..', 'core', 'src', 'codemods')
+  : path.join(process.cwd(), 'node_modules', '@compiled', 'core', 'dist', 'codemods');
```

Comment:
what would happen if we're running in a monorepo (with and without workspaces?)

Reply:
**With**: It will pick `node_modules/@compiled/core/dist/codemods` from  root folder. If anyone is running inside any package, it will still work because of symlink to `@compiled/react` in root folder.

**Without**: In this case, packages will not be hoisted to root `node_modules` and this cli will install `@compiled/react` to the folder from from where it is running.

Q: What will happen if we use `npx @compiled/cli`? `process.cwd()` will be temp directory where `@compiled/cli` will be installed or the path where we are executing `npx @compiled/cli`? 🤔

Reply:
ideally it should work with `npx` because not many people would want to install the cli directly - it would be like use once and throw away

will npx work atm?